### DISCUSSION TITLE
分享一下 W11的小技巧跟更新後的使用體驗

### DISCUSSION THREAD

**Discussion by Isongzhe** [1101-GS4719-C/Python-Community-GS4719-C#170] [General]
https://youtu.be/Z2HYPD_XF40
這是我之前發現的影片，我發現它裡面講的功能都蠻實用的
心得: 
前陣子我就更新成w11了，整體使用下來。最直接的感受就是整個系統變簡潔許多，並且把w10許多用不到又雞肋的東西移除掉，例如w10的工具列裡的一堆格子。但我認為現在的版本還不太穩定，不時就會有更新檔，建議可以再等等

**Comment by enoch-lightning** [comment]
我的電腦都沒有跳出要更新成windows 11的通知，所以我就沒有將windows 10更新成windows 11。

**Comment by chun1203** [comment]
已更新，謝謝分享相當的實用！

**Comment by EricAgumonJou** [comment]
很實用～電腦能玩手機遊戲這點很吸引我（還是喜歡使用大螢幕🤣），不知道是否是所有的手機遊戲都可以玩，還是只有特定的遊戲才有支援。

**Comment by xCk27x** [comment]
更新到 Win11 後有用到 To Do 跟時鐘的新功能
用來做備忘錄還有管理讀書時間
因為是直接跟作業系統整合的 所以用起來很流暢
真的很不錯 推薦一下

**Comment by enoch-lightning** [comment]
![image](https://user-images.githubusercontent.com/86505454/144350046-5f557887-0ce1-43fe-ae94-560b58684601.png)
我終於也收到Windows10可以更新成Windows11的通知。只是一看到
注意：某些Windows10的功能在Windows11中不提供，而部分應用程式及功能則可能會有其他需求。
就猶豫了。我怕會不會將電腦的Windows10作業系統，更新，也就是升級成Windows11的作業系統後，會不會Python的環境變數要重設，anaconda不能跑。之前在「程式設計—Python」課堂上安裝的原生Python，與anaconda會不會從電腦中消失，或是在之前安裝原生Python與anaconda時，在安裝過程中選取的設定都會消失。或是有一些電腦之前安裝或內建的東西，不見，不能跑(「跑」」就是「執行」)，或要重新設定，或是會與Window11作業系統有衝突。
這是不太熟悉電腦操作的我的想法，如果有人懂，包含老師與同學，就麻煩您們解惑。謝謝。

**Reply by enoch-lightning** [reply]
💻 🖱️ 🖥️ 💻 🖥️ 🖱️ 🖱️ 💻 🖥️ 🖱️ 🖥️ 💻 🖥️ 💻 🖱️ 🖥️ 🖱️ 💻 
 3!=3x2x1=6種排列組合。

**Comment by HSU-YU-JEN** [comment]
謝謝分享~但我好像被一些防毒軟體擋到，導致一直沒有寄更新通知過來
<img width="515" alt="image" src="https://user-images.githubusercontent.com/90545550/144700609-414d4a0d-deed-474a-87f1-689d769b99a3.png">

**Comment by CCello** [comment]
Windows 11確實有許多新功能和改進，並且在視覺設計上也進行了調整，讓整體感覺更簡潔和現代。然而，軟體的穩定性對於操作系統來說是非常重要的，如果還不夠穩定，建議可以再觀察一段時間或等待更多的更新。

**Comment by ashleyy-huang** [comment]
從更新了Windows 11之後，我覺得使用方面非常順暢，新增了許多小功能，像是CMD、檔案總管、文字文件有分頁設計，還有在桌面按右鍵後的選單變得更清晰簡單，在使用者角度是比較方便的，期待未來還會再有其他新奇的功能出現!

**Comment by BOSENCHANG2004** [comment]
以下是官網提供的特色:
1. 快速分割視窗（Snap Layouts）
將滑鼠游標移到視窗右上方的「最大化」按鈕，就會出現視窗排列選項，幫你快速分割工作區域。
2. 自訂開始功能表
到「設定 > 個人化 > 開始」，調整你想看到的應用程式或資料夾，避免雜亂。
3. 快速叫出表情符號與 GIF
使用快捷鍵 Win + . (Windows + 點號)，快速叫出表情符號、特殊字元和 GIF，非常方便。
4. 焦點模式（Focus Sessions）
到「時鐘」應用程式內開啟「專注模式」，可有效幫助自己專心工作或讀書，減少干擾。
